eemeter/modeling/models/hourly_model.py

```diff
@@ -64,14 +75,26 @@ def add_cdd(self, df):
         df_with_cdd = df.assign(cdd=cdd)
         return df_with_cdd
 
-    def print_model_stats(self,
-                          model_res):
+    def get_model_stats(self,
+                        model_res, df):
         if not model_res:
-            return
+            return {}
         rmse = np.sqrt(model_res.ssr/model_res.nobs)
-        print("Intercept: ", model_res.params['Intercept'],
-              "R2 ", model_res.rsquared_adj,
-              "RMSE :", rmse)
+        cvrmse = rmse / df['energy'].mean()
+        nmbe = np.nanmean(model_res.resid) / df['energy'].mean()
+
+        result = {"intercept": model_res.params['Intercept'],
```

Comment:
Would be good to pick one style of quotation and stick with it except in instances in which a quote would otherwise need to be escaped. I suggest sticking with single quotes (`'`).

Reply:
Changed

Reply:
Also changed the class name to HourlyDayOfWeekModel.